src/site/artifacts/cargo_dist.rs

```diff
@@ -5,6 +5,7 @@ use cargo_dist_schema::{Artifact, ArtifactKind, DistManifest, Release};
 
 use crate::config::Config;
 use crate::errors::*;
+use crate::site::changelog::fetch_releases;
```

Comment:
i think we should import changelog, not the function name

Reply:
updated all the imports there